### ISSUE TITLE
file and line information in log messages is not useful

### ISSUE THREAD

**Issue by okartau** [intel/pmem-CSI#174]
Currently used log messages location-stamping is not useful as the file:line information points to lines in glog.go:
Example: there different errors are shown at glog.go:119:
E0216 glog.go:119] GRPC error: rpc error: code = InvalidArgument desc = Volume ID missing in request
E0216 glog.go:119] GRPC error: rpc error: code = Internal desc = Unmount failed: exit status 32
E0216 glog.go:119] GRPC error: Device not found with name b7b5b605-320b-11e9-9a22-1a242cacdf22

**Comment by pohly** [comment]
I can take this issue and update the logging to use klog, as in upstream Kubernetes.

However, the problem with those log lines is that they get generated from deep inside gRPC infrastructure. There's no meaningful relation to the location in the code where the error was originally generated. If the error itself comes with an error stack, then it may be possible to include that information in the output. I'll check...

**Comment by okartau** [comment]
I also came to same conclusion after brief look. It's convenient to give error msg at gRPC return line (current style).
To get line number into log, it would take separate dedicated (and thus repeating the message) log line, right, so perhaps it's not justified to add it everywhere.
But nevertheless, I created this issue to have some discussion and ideas shared, after we had looked to some e2e logs with avalluri and noted that file:line info is not useful.
